lib/header.js

```diff
@@ -1,10 +1,38 @@
-import assert from 'assert';
+import moment from 'moment';
+import renderEmbed from './embed';
 import renderTextItems from './text-items';
 
-export default function (headerType, children) {
-  assert(headerType >= 1 && headerType <= 6);
+const componentsWithoutEmbed = ({title, author: {name, href}, date}) => [{
+  role: 'title',
+  text: title,
+  textStyle: 'titleStyle'
+}, renderTextItems(
+    'byline', [{
+      type: 'text',
+      content: 'By '
+    }, {
+      type: 'text',
+      content: name,
+      href: href
+    }, {
+      type: 'text',
+      content: ` ${moment(date).format('MMMM D, YYYY')}`
```

Comment:
yeah, let's rename to publishedDate. There are other types of dates we will have to pass too.